### PR TITLE
docs: fix docs links

### PR DESCRIPTION
**Pull request by Hugoo**
We had a couple of broken links

### CODE REVIEW

**Comment by Hugoo** [comment]
When this is merged we can re sync our main docs:

https://github.com/lukso-network/docs/pull/564

**Review by CallumGrindle — COMMENTED** [review]
this is reverting the changes @YamenMerhi made updating the LSP3 docs. Can we fix on the docs rather than revert correct changes here?

File: docs/classes/universal-profile.md [hunk 1]
```diff
@@ -320,9 +322,10 @@ Available as a static or non-static method callable on the LSPFactory library in
 
 #### 1. `profileData` - Object
 
-Object containing the [LSP3 Metadata](../../../standards/universal-profile/lsp3-profile-metadata) fields which will be processed and uploaded to IPFS.
+Object containing the [LSP3 Metadata](../../../standards/universal-profile/lsp3-universal-profile-metadata.md) fields which will be processed and uploaded to IPFS.
```
Comment (CallumGrindle) [inline docs/classes/universal-profile.md:325]: These ones were updated by @YamenMerhi here https://github.com/lukso-network/tools-lsp-factory/pull/198/files
Can we remove these reverts here? These should be correct

Comment (Hugoo) [reply docs/classes/universal-profile.md:325]: I don't see the initial file in the docs repo, I only see this one:

https://github.com/lukso-network/docs/blob/main/docs/standards/universal-profile/lsp3-universal-profile-metadata.md

Comment (Hugoo) [reply docs/classes/universal-profile.md:325]: image

**Comment by Hugoo** [comment]
ah i see, this PR: https://github.com/lukso-network/docs/pull/593/files

File: docs/classes/universal-profile.md (hunk 1 above)
Comment (Hugoo) [reply docs/classes/universal-profile.md:325]: Nice catch, let's merge the other PR first then:

https://github.com/lukso-network/docs/pull/593/files

Comment (Hugoo) [reply docs/classes/universal-profile.md]: Fixed

**Review by CallumGrindle — APPROVED** [review]

**Pull request merged**